java-security/src/main/java/com/sap/cloud/security/servlet/AbstractTokenAuthenticator.java

```diff
@@ -154,8 +154,19 @@ protected TokenAuthenticationResult authenticated(Token token) {
 		return TokenAuthenticatorResult.createAuthenticated(Collections.emptyList(), token);
 	}
 
-	private boolean headerIsAvailable(String authorizationHeader) {
+	boolean headerIsAvailable(String authorizationHeader) {
 		return authorizationHeader != null && !authorizationHeader.isEmpty();
 	}
 
+	TokenAuthenticationResult tokenValidationResult(Token token){
```

Comment:
this can be used within the same class
https://github.com/SAP/cloud-security-xsuaa-integration/blob/08abdd4a0b2dfe5bcca8cfab6e76ca5c415525e1/java-security/src/main/java/com/sap/cloud/security/servlet/AbstractTokenAuthenticator.java#L43

Reply:
@nenaraab now I have, thanks :)